lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -24,7 +24,7 @@
 <script type="text/javascript">
     (function (require) {
         require(['teams/js/teams_tab_factory'], function (TeamsTabFactory) {
-            new TeamsTabFactory(${ json.dumps(topics, cls=EscapedEdxJSONEncoder) }, '${ topics_url }', '${ unicode(course.id) }');
+            new TeamsTabFactory(${ json.dumps(topics, cls=EscapedEdxJSONEncoder) }, '${ topics_url }', '${ teams_url }', ${ course.teams_max_size }, '${ unicode(course.id) }');
```

Comment:
Wrap this so that it is easier to see how many arguments there are. In fact, I would be tempted to make the factory take a single options dict to make it easier to read.